tests/test_symbolic_primitives.py

```diff
@@ -209,39 +212,51 @@ def test_place_ontop(self, env, prim_gen, apple, pan):
             env.step(action)
         assert apple.states[object_states.OnTop].get_value(pan)
 
-    @pytest.mark.skip(reason="primitives are broken")
-    def test_toggle_on(self, env, prim_gen, stove):
+    def test_toggle_on(self, env, prim_gen, stove, sink):
         assert not stove.states[object_states.ToggledOn].get_value()
         for action in prim_gen.apply_ref(SymbolicSemanticActionPrimitiveSet.TOGGLE_ON, stove):
             env.step(action)
         assert stove.states[object_states.ToggledOn].get_value()
 
-    @pytest.mark.skip(reason="primitives are broken")
-    def test_soak_under(self, env, prim_gen, robot, sponge, sink):
-        water_system = env.scene.get_system("water")
-        assert not sponge.states[object_states.Saturated].get_value(water_system)
-        assert not sink.states[object_states.ToggledOn].get_value()
+        # toggle off the stove after the test is done
+        for action in prim_gen.apply_ref(SymbolicSemanticActionPrimitiveSet.TOGGLE_OFF, stove):
+            env.step(action)
+        assert not stove.states[object_states.ToggledOn].get_value()
```

Comment:
This is not a good way of doing this - if the test fails in the assertion above this one, these will not execute, and thus the next test will inherit dirty state. Find some other way via the fixture.

Reply:
Recommendation: remove this